src/templates/assignEmail.js

```diff
@@ -0,0 +1,78 @@
+export const renderAssignEmail = (plate, link) => {
+	const template = `<!doctype html>
```

Comment:
hey @Yvad60 , could you add a screenshot of this email template?

Reply:
For the undefined part it because I have no env variable for the LOGIN_URL